### ISSUE TITLE
Some symbols fail if not using UTF-8 input

### ISSUE THREAD

**Issue by vbitsch** [josephwright/siunitx#470]
After upgrade to version 3, \ang dont work anymore.

**Comment by josephwright** [comment]
Can you provide an example? For me
```latex
\documentclass{article}
\usepackage{siunitx}
\begin{document}
\ang{1;2;3} or \ang{1.23}
\end{document}
``` 
is behaving correctly.

**Comment by vbitsch** [comment]
Hi Joseph

It seems to be when I want to use the Danish letters æ, ø, and å.

\documentclass{article}

\usepackage[latin1]{inputenc}
\usepackage[T1]{fontenc}

\usepackage{siunitx}
\begin{document}
\ang{1;2;3} or \ang{1.23}
\end{document}
Med venlig hilsen

Viggo Bitsch

Mobil 26 84 78 80
E-Mail: ***@***.******@***.***>

Fra: Joseph Wright ***@***.***>
Sendt: 4. juni 2021 16:32
Til: josephwright/siunitx ***@***.***>
Cc: Viggo Bitsch ***@***.***>; Author ***@***.***>
Emne: Re: [josephwright/siunitx] Angle sign (#470)


Can you provide an example? For me

\documentclass{article}

\usepackage{siunitx}

\begin{document}

\ang{1;2;3} or \ang{1.23}

\end{document}

is behaving correctly.

**Comment by josephwright** [comment]
Fixed by 1f373d82c09c66c98e98d2d9ea6e208e6c2c0fc5
